apps/wallet/src/ui/app/staking/stake/StakeForm.tsx

```diff
@@ -141,7 +141,10 @@ function StakeForm({
                                 weight="medium"
                                 color="steel-darker"
                             >
-                                {unstake ? 0 : calculateRemaining} {symbol}
+                                {calculateRemaining <= 0
```

Comment:
small nit - 

no space between value and symbol here 
![image](https://user-images.githubusercontent.com/122397493/214440223-510128a9-45b8-400d-b97a-516ae03cafbf.png)